extensions/security/deployment/src/main/java/io/quarkus/security/deployment/DotNames.java

```diff
@@ -7,11 +7,13 @@
 import org.jboss.jandex.DotName;
 
 import io.quarkus.security.Authenticated;
+import io.quarkus.security.PermissionsAllowed;
 
 public final class DotNames {
 
     public static final DotName ROLES_ALLOWED = DotName.createSimple(RolesAllowed.class.getName());
     public static final DotName AUTHENTICATED = DotName.createSimple(Authenticated.class.getName());
+    public static final DotName PERMISSIONS_ALLOWED = DotName.createSimple(PermissionsAllowed.class.getName());
```

Comment:
`quarkus-security`  has not been released yet, so the question, can we have this annotation named as `Permissions`, with `allowed` being implicit, or would you like to keep an option of eventually having `PermissionsDenied` ?

Reply:
I think `Permissions` is better for brevity, but that is very subjective and in order to get avoid conflicts, I just tried to stick as close as suggestions in the issue (almost entirely from @stuartwdouglas ) as possible. But permissions denied is very interesting idea which maybe in the future, users might as for. I'd stick with allowed as it follows `RolesAllowed`, but honestly, whatever :-)